rbc/tests/test_omnisci_column_basic.py

```diff
@@ -919,3 +919,49 @@ def convolve(x, kernel, m, y):
 
     expected = [(10.0,), (60.0,), (180.0,), (240.0,), (300.0,)]
     assert list(result) == expected
+
+
+@pytest.fixture(scope='function')
+def add_nulls(omnisci):
+    omnisci.sql_execute(
+        f'insert into {omnisci.table_name} '
+        f'values (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);'
+    )
+    yield omnisci
+    omnisci.sql_execute(
+        f'delete from {omnisci.table_name} where f4 is NULL;'
+    )
+
+
+colnames = ['f4', 'f8', 'i1', 'i2', 'i4', 'i8', 'd']
+types = ['float', 'double', 'int8', 'int16', 'int32', 'int64', 'double']
+
+
+@pytest.mark.skipif(
+    available_version < (5, 5),
+    reason=(
+        "test requires omniscidb with type_limits"
+        " support (got %s) [issue 188]" % (
+            available_version,)))
+@pytest.mark.usefixtures('add_nulls')
+@pytest.mark.parametrize('col,typ', zip(colnames, types))
+def test_null_value(omnisci, col, typ):
+    omnisci.reset()
+    omnisci.register()
+
+    @omnisci(f'int32(Column<{typ}>, RowMultiplier, OutputColumn<{typ}>)')
+    def my_row_copier_mul(x, m, y):
+        input_row_count = len(x)
+        for i in range(input_row_count):
+            if x.is_null(i):
+                y.set_null(i)
+            else:
+                y[i] = x[i] + x[i]
+        return m * input_row_count
```

Comment:
should we move this test to `test_omnisci_column_null.py

Reply:
actually, this is a duplicate of the test in test_omnisci_column_null.py. I suggest removing the test here.